x-pack/plugins/apm/common/alert_types.ts

```diff
@@ -53,18 +76,27 @@ export function formatTransactionDurationReason({
   measured,
   serviceName,
   asDuration,
+  aggregationType,
+  windowSize,
+  windowUnit,
 }: {
   threshold: number;
   measured: number;
   serviceName: string;
   asDuration: AsDuration;
+  aggregationType: string;
+  windowSize: number;
+  windowUnit: string;
 }) {
   return i18n.translate('xpack.apm.alertTypes.transactionDuration.reason', {
-    defaultMessage: `Latency is above {threshold} (current value is {measured}) for {serviceName}`,
+    defaultMessage: `{aggregationType}. latency is {measured} in the last {interval} for {serviceName}. Alert when > {threshold}.`,
```

Comment:
@fkanout is the `.` right after `{aggregationType}` correct? I guess aggregationType holds a sentence, I just want to make sure it is not a typo.

Reply:
@mgiota, thanks for pointing that out. I used the first use case in the table as a reference (with Avg aggregation type). but the point there seems for the word `Avg` abbreviation. I will fix that. (add the point only when Avg)